ui-particles-angular/projects/ui-policy-studio-angular/src/lib/components/flow-execution-form-dialog/gio-ps-flow-execution-form-dialog.component.html

```diff
@@ -0,0 +1,56 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<h2 mat-dialog-title class="title">
+  Flow execution
+
+  <button class="title__closeBtn" mat-icon-button aria-label="Close dialog" [mat-dialog-close]="false">
+    <mat-icon svgIcon="gio:cancel"></mat-icon>
+  </button>
+</h2>
```

Comment:
we could make a component, but it must not continue the `mat-dialog-close`. 
But I haven't taken the time to make a component for that. I don't think it would be very valuable in relation to the time spent making it. or later, during a refactoring phase. 
There is a lot to do and here to avoid 15 lines and still because make a component it makes by default 3 file + 20 line by default I do not know. 
when we've finished all the dialogues I figure it'll be a good time to think about it ;)